app/src/main/java/com/fynov/equaleyes/appestimator/ui/viewholders/CategoryViewHolder.java

```diff
@@ -0,0 +1,36 @@
+package com.fynov.equaleyes.appestimator.ui.viewholders;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import com.fynov.equaleyes.appestimator.data.models.Category;
+import com.fynov.equaleyes.appestimator.data.models.Feature;
+import com.fynov.equaleyes.appestimator.databinding.ItemCategoryBinding;
+import com.fynov.equaleyes.appestimator.ui.adapters.FeatureAdapter;
+
+public class CategoryViewHolder extends RecyclerView.ViewHolder {
+    private ItemCategoryBinding mBinding;
+    FeatureAdapter mAdapter;
+    public CategoryViewHolder(ItemCategoryBinding binding) {
+        super(binding.getRoot());
+        mBinding = binding;
+    }
+
+    public void bind(@NonNull Category category) {
+
+        int sum = 0;
```

Comment:
Bind se kliče ko se viewholder prikaže. Ali se skupni čas posodobi, če je viewholder že prikazan?

Reply:
Za enkrat se ne, to se morm ose pogruntat kak bo retroaktivno updajtalo.